ffc-node-base/Dockerfile

```diff
@@ -0,0 +1,19 @@
+ARG NODE_VERSION=10.18.0
+FROM node:$NODE_VERSION-alpine
+ARG NODE_VERSION

```

Comment:
Duplicate `ARG NODE_VERSION`

Reply:
ARG's declared before the FROM line are not available to the stage unless redeclared below the FROM line

Reply:
Ah of course!